### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by NATASHA-ct**
- [ ] Modified my html and css files due to broken code.
- [ ] Added a footer with a contact form.
- [ ] Completed the mobile version.

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @NATASHA-ct, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,271 +1,282 @@
...
 
-        <textarea
-          class="message"
-          name="message"
-          id="message"
-          maxlength="500"
-          placeholder="Enter text here"
-          required></textarea>
+  
+        <textarea class="textarea1" name="message" id="textarea1" maxlength="500" placeholder="Enter text here" required></textarea>
```
Comment (vechicin) [inline index.html:236]: As stated by the previous reviewer, placeholder texts need to be meaningful. Something like `Enter name`, `Enter email` and `Type message here` will suffice. Please fix this.

Screen Shot 2022-06-07 at 12 23 31

File: styles.css [hunk 2]
```diff
@@ -356,175 +324,118 @@ button.see-prj:disabled {
...
-  width: 327px;
-  flex: none;
-  flex-grow: 0;
+/* form */
+.form {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 24px;
+  order: 1;
 }
```
Comment (vechicin) [inline styles.css:379]: According to the project requirements, this entire section needs to be aligned using flexbox. Kindly fix this.

Screen Shot 2022-06-07 at 12 25 39

File: styles.css [hunk 3]
```diff
@@ -326,17 +298,13 @@ h2.project-title {
...
-  width: 89px;
-  height: 48px;
   background-color: #7d73fc;
   color: #ecebff;
-  margin-left: 30px;
+  width: 89px;
+  height: 48px;
   border: none;
+  order: 3;
 }
```
Comment (vechicin) [inline styles.css:308]: The button is not placed correctly in your design.

Your design:

Screen Shot 2022-06-07 at 12 27 45

Template design:

Screen Shot 2022-06-07 at 12 28 11

Please fix this as it is a project requirement.

Screen Shot 2022-06-07 at 12 28 35

File: index.html [hunk 4]
```diff
@@ -1,271 +1,282 @@
...
+                  <!--vector line-->
+     <div class="short">
+        <img class="horizontal-line" src="./background-shapes/mobile-version/Vector 4.png" alt="horizontal-line">
+      </div>
+  
+  
+    <!--social media icons-->
+  
+    <div class="bottom-footer">
+      <div class="social footer-icons">
```
Comment (vechicin) [inline index.html:254]: Your icons don't quite look like the ones from the template design.

Your design:

Screen Shot 2022-06-07 at 12 29 55

Template design:

Screen Shot 2022-06-07 at 12 30 19

I recommend you download them directly from the Figma template. Please fix this as it is a project requirement.

Screen Shot 2022-06-07 at 12 30 56

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @NATASHA-ct ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Organized code
- [x] Correct GitHub Flow
- [x] Good PR title/description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ginohmk — APPROVED** [review]
Hi @NATASHA-ct , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**